src/main/webapp/user_profile.html

```diff
@@ -6,40 +6,73 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getUserInfo()">
     <div id="navigation">
       <a href="main_hub.html" id="hub">Main Hub</a>
       <a href="user_profile.html" id="user">User Profile</a>
       <a href="project.html" id="project">Project Page</a>
       <a href="task.html" id="task">Task Page</a>
     </div>
     <div id="content">
-      <h1>Welcome, UserName</h1>
+      <h1 id="user-name-container">Default User Name</h1>
       <h2>User Stats:</h2>
-      <div id="user-stats">
-        <div>Name: UserName LastName</div>
-        <div>Year: Junior</div>
-        <div>Major: Biology</div>
-        <div>Total Completed Tasks: 7</div>
-        <button type="button" onclick="goToTask()">See Tasks</button>
+      <div class="user-stats">
+        <div id="year-container">Year: Default Year</div>
+        <div id="major-container">Major: Default Major</div>
+        <div id="num-complete-container">Total Completed Tasks: default</div>
       </div>
-      <h2>Checklist:</h2>
+      <h2>User Skills:</h2>
+      <div class="user-stats">
+        <h3 id="skills-container">Default Skillset</h3>
+      </div>
+      <!--Only show this if the user is looking at their own profile-->
+      <h2>Change Settings: </h2>
+      <div class="user-settings">
+        <h3>Change Skills:</h3>
+        <form>
+          <input type="checkbox" id="leadership" name="leadership" value="leadership">
+          <label for="leadership">Leadership</label><br>
+          <input type="checkbox" id="organization" name="organization" value="organization">
+          <label for="organization">Organization</label><br>
+          <input type="checkbox" id="writ" name="writ" value="writ">
+          <label for="writ">Analytical/Creative Writing</label><br>
+          <input type="checkbox" id="art" name="art" value="art">
+          <label for="art">Artistic Ability</label><br>
+          <input type="checkbox" id="webdev" name="webdev" value="webdev">
+          <label for="webdev">Web Development (HTML,CSS,Javascript)</label><br>
+          <input type="checkbox" id="oop" name="oop" value="oop">
+          <label for="oop">Object Oriented Programming Languages (ex:java)</label><br><br>
```

Comment:
ok for the prototype, I would suggest not to use br for formatting your page though, since semantically br would be more for breaking a line of text into multiple lines.  Maybe ul/li would be more appropriate.